main/guides/orchestration/dapp-orchestration-basics/installation/index.md

```diff
@@ -0,0 +1,218 @@
+
+To use this dapp with the kubernetes environment spun-up by `multchain-testing` & starship, we have to make sure vstorage reflects the chain information relevent to our locally-maintained environment. 
+
+Run the following commands from the `agoric-sdk/multichain-testing/` directory:
+
+```bash
+make override-chain-registry
+```
+
+## Multichain-Testing Makefile Helpers
+To streamline your workflow, you can add the following commands to the bottom of your `multichain-testing/Makefile`:
+
+```makefile
+teardown: stop-forward stop clean delete
+
+corepack-setup:
+	corepack prepare yarn@4 --activate
+
+corepack-enable:
+	corepack enable
+
+test:
+	yarn test test/install-contracts.test.ts
+
+all: setup install
+	sleep 3
+	make port-forward
+	sleep 120
+	make fund-provision-pool
+	sleep 10
+	make add-address
+	echo "done running"
+
+hermes-update:
+	kubectl exec -i hermes-agoric-osmosis-0 -c relayer -- hermes update client --host-chain agoriclocal --client 07-tendermint-1 # or 07-tendermint-0
+	sleep 60
+	make hermes-update
+```
+
+**Restart the Environment**  
+If you need to restart your environment for any reason, use the following command from `agoric-sdk/multichain-testing`:
+```bash
+make teardown ; make stop; make stop-forward; make clean; make; make port-forward
+```
+
+## Troubleshooting Docker
+During setting up the environment, you may run into a few issues with docker. Here are two common setup requirements.
+
+### Enabling Kubernetes
+Inside of "settings", navigate to the "kubernetes" section. Here, you want to ensure that you select the "Enable Kubernetes" checkbox. You will have to restart docker to make these changes take effect. This is something to keep this in mind if you have any containers that are currently running.
+
+<img src="../../assets/docker-kube.png" width="100%" />
+
+### Increasing resources allocated to docker
+From inside of "settings", navigate to the "Resources" section. By default docker doesn't optimize the resources allocated for running resource-intensive environments. For this environment, we want to ensure our `CPU limit`, `Memory Limit`, and `Swap` are all bumped up to 75% their max. You can always increase/decrease after you get everything setup, and determine how much resources are needed for your machine.
+
+<img src="../../assets/docker-resources.png" width="100%" />
+
+
+## Adding a New Address
+
+Before deploying the contract, we want to generate a new address using the keyring inside of the localchain environment. To add a new address to the keychain inside the Kubernetes pod, run the following command from the top-level directory:
+
+```bash
+make add-address
+```
+
+Now, inside of `contract/`, paste the resulting `agoric1` prefixed address into the `Makefile` under the `ADDR`variable.
+
+## Funding the Account
+
+To now fund the account we just generated, we run the following command from the top-level directory. 
+
+```bash
+make fund
+```
```

Comment:
I am unable to proceed past this step. Which Makefile has this `fund` target? I can only see `fund-wallet` under `multichain-testing` and `fund-acct` under `dapp-orchestration-basics/contract`

Reply:
good q @mujahidkay , this is from `dapp-orchestration-basics` top-level directory:
https://github.com/Agoric/dapp-orchestration-basics/blob/d43ebe67012a85ac404d76687bf91f9571b10594/Makefile#L9

which is just calling:
```console
yarn workspace  dapp-agoric-orca-contract fund
```

which then just calls the fund as convenience:
https://github.com/Agoric/dapp-orchestration-basics/blob/d43ebe67012a85ac404d76687bf91f9571b10594/contract/package.json#L26